### PR TITLE
fix: blobfuse mount failure in Azure Linux v3

### PR DESCRIPTION
**Pull request by cvvz**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix: mout failed by blobfuse2 in azure linux v3

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:
azurelinux v2(mariner) node:
image

azurelinux v3 node:
image

**Release note**:
```
fix: mout failed by blobfuse2 in azure linux v3
```

### CODE REVIEW

File: pkg/blobfuse-proxy/server/server.go [hunk 1]
```diff
@@ -121,8 +121,8 @@ func getBlobfuseVersion() BlobfuseVersion {
 		return BlobfuseV1
 	}
 
-	if strings.EqualFold(osinfo.Distro, "mariner") && osinfo.Version >= "2.0" {
-		klog.V(2).Info("proxy default using blobfuse V2 for mounting on Mariner 2.0+")
+	if (strings.EqualFold(osinfo.Distro, "mariner") || strings.EqualFold(osinfo.Distro, "azurelinux")) && osinfo.Version >= "2.0" {
```
Comment (andyzhangx) [inline pkg/blobfuse-proxy/server/server.go:124]: could you paste the details of `osinfo.Distro` in comments, thx

Comment (cvvz) [reply pkg/blobfuse-proxy/server/server.go:124]: This variable is read from `/etc/os-release` and I have pasted the screenshoot of azurelinux v2 and v3 node in pr description.

**Review by andyzhangx — APPROVED** [review]
/lgtm

**Pull request merged**